config/archetype/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>org.kuali.rice</groupId>
     <artifactId>rice-config</artifactId>
-    <version>ua-2.5-release1</version>
+    <version>2.5.19-uar11-SNAPSHOT</version>
```

Comment:
Yeah, the previous ones - i.e. ua-release10 - probably refer to the UAF6 version of rice - I also like the shorter version "2.5.19-ua1" since it is unambiguous about the rice version. Since it relates to rice 2.5.19 it is the first of its kind... (not the 11th one)